Middleware/SecurityMiddleware.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace League\Tactician\Bundle\Middleware;
+
+use League\Tactician\Exception\InvalidMiddlewareException;
+use League\Tactician\Middleware;
+use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+
+class SecurityMiddleware implements Middleware
+{
+    /**
+     * Access denied behavior to drop the command if not allowed.
+     */
+    const DROP_COMMAND = 1;
```

Comment:
If I read it correctly, drop command mode fails silently when the command is forbidden? If that's the case, I'd prefer to do without this entirely: even though it's not the default, I feel like this sort of thing should hit the log or be flagged somewhere, etc. It's fairly significant security failure if it already made it past the UI layer to here.

If someone really needs this for legacy or another approach, you've slimmed down the voter enough that it should be really easy for them to make their own silencing version. :)